index.js

```diff
@@ -31,6 +31,7 @@ export default function (kibana) {
         description: "Kibana plugin for Index Management",
         main: "plugins/opendistro_index_management_kibana/app",
         category: DEFAULT_APP_CATEGORIES.management,
+        order: 9010,
```

Comment:
Can you add a comment why this is set to 9010 (i.e. because Stack Management is ### and DevTools is ####) and/or add a const with a more meaningful name

Reply:
sure